config/game_options.txt

```diff
@@ -400,10 +400,11 @@ ROUNDSTART_RACES human
 
 ## Races that are okay-ish on the station. Safe to enable.
 ROUNDSTART_RACES unathi
-ROUNDSTART_RACES plasmaman
-ROUNDSTART_RACES slime
+#ROUNDSTART_RACES plasmaman
+#ROUNDSTART_RACES slime
 ROUNDSTART_RACES ethari
-ROUNDSTART_RACES pod
+#ROUNDSTART_RACES pod
```

Comment:
Goofy that this is here, as thats what is live.

Reply:
Maybe we should move them to the section below instead of living them here just commented out 🤔